### ISSUE TITLE
Consider searching for `@isTest` to automatically add test classes found

### ISSUE THREAD

**Issue by mcarvin8** [renatoliveira/apex-test-list#45]
I thought about this while using this plugin in our repo. We use a manifest file to just deploy what we change, via the sfdx-git-delta plugin.

In the event there is a manifest file with just test classes, if there isn't test annotations in there, our pipeline would fail since we enforce Specified Tests rather than running all local tests to save time.

Our developers have onboarded to using the Test Annotations method in the main classes and triggers, but they do not always add them to the test classes.

What I'm suggesting is to add code which:
1. First searches for the `@isTest` string which indicates if it's a test class or not. If true, append that file-name to the final output.
2. If `@isTest` is not True (for main classes and triggers), search for the `@Tests:` annotation in the file

**Comment by renatoliveira** [comment]
So you mean that for a scenario where a change is applied only on test classes, these should be detected also, not just the annotated ones? Like fixing a unit test, for example? 

Would this search happen only on changed files by default, and on the manifest also when it is provided through the flag?

**Comment by mcarvin8** [comment]
Yes essentially that, when a test class is the only update made, it should use the test class name. Like if someone just had to fix a unit test.

It definitely would apply to the manifest flag if the package only contained the test class.

Perhaps the 1st update would be:
1. When manifest flag is used, search for `@isTest` to determine if it's a test class and if yes, use that name instead of searching for your annotations. 

We could think more about this in terms of the entire package directory. Not sure what's the best path for that.

Also, unrelated, but noticed this with some of our developers. It would be helpful if all of the annotations this plugin searches for `@Test:` are case insensitive. Saw an issue when a developer used `@test:` in their annotation. If you add code to look for the `@isTest` annotation in test classes, this should be case insensitive as well.

**Comment by renatoliveira** [comment]
Closing this because it has been added in 1.6.0 now.
